### DISCUSSION TITLE
Unable to verify my proof for Student Pack (remote learning)

### DISCUSSION THREAD

**Discussion by realsunyz** [community/community#144167] [GitHub Education]
### Hi everyone,

I am a remote learning high school student, and our school does not provide students with an edu email address or any other official letter or student ID that can be used for verification.

However, the school uses Canvas LMS to provide courses and manage grades. I am currently enrolled in several courses and provided a screenshot of the LMS interface with the school logo as proof of my enrollment. For my screenshot, I chose the “Grades” page for the courses I am taking this year, which includes my name, the course title, when I submitted the assignment (October and November of this year), and the teacher's grade. I believe this proves that I am a currently enrolled student.

Current problems:
- **Screenshot of the grades page**: the system says, “Your image contains keywords indicating that it may be an acceptance letter. Only proof of current academic enrollment is acceptable.” I'm confused because my screenshot is not an acceptance letter, but rather one with in-semester course grades.
- **Screenshot of the assignment submission page**:  I then tried another screenshot, choosing an assignment submission page from the end of October, which contained my name, submission date, teacher's comments, comment date, and school logo. However, this time, the system replied, “Unfortunately, we were unable to verify your academic affiliation based on the information you submitted,” without any specific reason.
- **Additional details**: All images are computer screens taken using a mobile phone with location permissions turned on and VPN turned off. As further proof of my remote learning status, I have also included a screenshot of the webpage on the school's website that shows the course program as remote learning.

I would like to know why I was rejected. Is there another way to verify my enrolled status?

Many thanks,
Seven

**Comment by Aditya7116** [comment]
"Do you have a copy of the admission transcript or the fee receipt available?"

**Reply by realsunyz** [reply]
> "Do you have a copy of the admission transcript or the fee receipt available?"

The system thinks my admission transcript is an acceptance letter, and I don't have a fee receipt. 😢

In addition, for physical proofs, all I have are exam papers with names, dates of writing, and school logos printed on them.
